src/extras/PMREMGenerator.js

```diff
@@ -230,9 +261,9 @@ PMREMGenerator.prototype = {
 			magFilter: NearestFilter,
 			minFilter: NearestFilter,
 			generateMipmaps: false,
-			type: equirectangular ? equirectangular.type : UnsignedByteType,
-			format: equirectangular ? equirectangular.format : RGBEFormat,
-			encoding: equirectangular ? equirectangular.encoding : RGBEEncoding,
+			type: this._type,
```

Comment:
This is the format of the renderTargets PMREM will write to and output. If this is `FloatType` above, it will fail on any device that can't write to float framebuffers, which I believe is why we've already had a report of black renders with this change. See related comment above.

Reply:
Yes, but the question is whether we want to kill the ability to render to float framebuffers or not. 
The way we're trying to set-up, we let the application decide that.

Reply:
Ah, okay, I finally understand the problem. I was confused because this PR changes several things at once. I believe that allowing the application to choose the data type of the output is a foot-gun because anything other than 8-bit will fail on many mobile devices. Also, in order to get any benefit from it, you would also need to change the shaders to handle this case without custom interpolation. 

I agree that you don't want to blindly follow the input data type for the output data type, so that change is good. The only reason it did that originally was to support both RGBE and sRGB output. But it would be a very simple change to just use RGBE always.

Reply:
> Also, in order to get any benefit from it, you would also need to change the shaders to handle this case without custom interpolation.

I think we can try that at a different PR, I could take a look at it. 
But the main benefit right now, in the case of EXR at least, is that we don't need to convert the data to RGBE on the CPU before inputting to PMREM. EXR doesn't have internal data as RGBE, it's always 16/32 float.

Reply:
Agreed; personally I'd remove the `setDataType` from this PR and put it in the follow-on if/when that happens. I'm still concerned about it being a foot-gun. The minimal change would be to use RGBE for any input encoding that's not sRGB. The other way is to make the internal format RGBE always. I'll defer to @WestLangley on that one. Just make sure to test a few JPG environments.